velero-2-mixin/targets.libsonnet

```diff
@@ -0,0 +1,96 @@
+local g = import './g.libsonnet';
+local prometheusQuery = g.query.prometheus;
+local commonlib = import 'common-lib/common/main.libsonnet';
+local lokiQuery = g.query.loki;
+local utils = commonlib.utils {
+  labelsToPanelLegend(labels): std.join(' - ', ['{{%s}}' % [label] for label in labels]),
+};
+{
+  new(this): {
+    local vars = this.grafana.variables,
+    local config = this.config,
+
+    succesfulBackups:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum(increase(velero_backup_success_total{%(queriesSelector)s}[$__interval:]))' % vars
```

Comment:
I think it makes sense to add `/ $__interval:` to the title for the panels using increase here.